app/util/settings.py

```diff
@@ -245,9 +261,11 @@ class Settings(BaseSettings, metaclass=SingletonBaseSettingsMeta):
     stripe: StripeConfig = stripe_config
     email: EmailConfig = email_config
     jwt: JwtConfig = jwt_config
-    database: DatabaseConfig = database_config
+    database: DatabaseConfig = database_config or DatabaseConfig(
+        url="sqlite:///:memory:"
```

Comment:
I am not a fan, if the config file it not found or misconfigured in prod, this could fail silently.

Reply:
Then we can add the logic above to include a `logging.warn` like I did for keycloak